integration_test.go

```diff
@@ -78,13 +78,55 @@ func TestCreateAzureResourcesIntegration(t *testing.T) {
 	autopilot.Equals(t, "new azure resources", result.Name)
 	autopilot.Equals(t, "12345678-1234-1234-1234-123456789abc", result.TenantId)
 	autopilot.Equals(t, "12345678-1234-1234-1234-123456789def", result.SubscriptionId)
-	autopilot.Equals(t, []string{}, result.Aliases)
+	autopilot.Equals(t, []string{"new_azure_resources"}, result.AzureResourcesIntegrationFragment.Aliases)
```

Comment:
Updated this test to check for aliases because it's required by GoLang since the integration contains both "fragment" structs that contain an attribute named `Aliases`.